class-00/lab-b/starter-code/app.js

```diff
@@ -184,17 +176,17 @@ Student.prototype.scope = function() {
 };
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scope());
+console.log(joe.scope());
 
 Student.prototype.scopeArrow = () => console.log(this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scopeArrow());
+console.log(joe.scopeArrow());
 
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
-//
+// "this" is the object itself, this does not have a value. This will get a value when a new object is created. You also cant change the value of this, its not a variable.
 // 2. What is "this" when joe.scopeArrow() is invoked?
-//
+// when you try to invoke this while using the arrow function instead of normally calling the function. You get undefined because the arrow function doesnt have a this. The arrow function only follows the normal variable look up rules.
```

Comment:
Arrow functions don't bind 'this' meaning that when you try to use 'this' inside an arrow function you won't get the object you're trying to!